content/how-to/dedicated-hosting/add-new-vpn-user-myrackspace-portal/index.md

```diff
@@ -5,66 +5,80 @@ title: Add a new VPN user in the MyRackspace Portal
 ---
 
 Adding new client VPN users is now an automated task within the MyRackspace
-Portal. This article describes how to add a new VPN user by using a ticket
-template.
+Portal. This article describes how to add a new VPN user by using the Support Center.
 
 ### Create a ticket in the MyRackspace Portal
 
-1. Log in to the [MyRackspace Portal](https://login.rackspace.com) by using your
+**Step 1.** Log in to the [MyRackspace Portal](https://login.rackspace.com) by using your
    username and password.
 
-2. In the top navigation bar, click **Select a Product > Rackspace Dedicated**.
+   <img width="409" alt="Enter the login credentials with Username and Password" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn1.png">
 
-3. Select **Tickets > Create New Ticket**.
+**Step 2.** In the top navigation bar, click **Support > Support Center**.
 
-4. On the **Create New Ticket** page, click the **Subject** field, and in
-   the drop-down menu, select **VPN User Management - add, reset, delete VPN user(s) on firewall(s)**.
+   <img width="941" alt="Select support center from the top navigation bar" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn2.png">
 
-   {{<image src="image001.jpeg" alt="" title="">}}
+**Step 3.** Under **Common Request**, click **See All Templates**. A list of all Common Request templates will be displayed.
 
-5. Check the box next to **Create new VPN user on firewall**.
+   <img width="958" alt="Displays the list of templates" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn3.png">
 
-6. In the **Firewall** drop-down field, select the appropriate firewall(s).
+**Step 4.** From the **Common Request** menu, select **VPN User Management - add, reset, delete VPN user(s) on firewall(s)**.
 
-   a. If you have high availability (HA) firewalls, you may select only one of the two
-      HA firewalls in the drop-down menu. Automation adds the user to both HA firewalls.
+   <img width="830" alt="Select VPN User Management- add, reset, delete VPN user(s) on firewall(s) from the menu" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn4.png">
 
-   b. If you need to add the VPN user to all firewalls on the account, ensure
+**Step 5.** In the **Issue Details** section you can select one, multiple or all of the following VPN user management options as shown in following image.
+
+   <img width="633" alt="Select any one or multiple VPN user on firewall" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn5.png">
+
+**Step 6.** Check the box next to **Create new VPN user on firewall**.
+
+   <img width="408" alt="Select the create new VPN option" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn6.png">
+
+**Step 7.** In the **Firewall** drop-down field, select the appropriate firewall(s).
+
+   <img width="370" alt="Select required firewall from the drop-down list" src="/support/how-to/add-new-vpn-user-myrackspace-portal/newvpn7.png">
```

Comment:
As we noticed, looks like device is showing in image.

Reply:
Image is replaced as suggested